tensorflow/python/training/saver.py

```diff
@@ -1276,7 +1276,7 @@ def restore(self, sess, save_path):
     if save_path is None:
       raise ValueError("Can't load save_path when it is None.")
 
-    if not checkpoint_management.checkpoint_exists(compat.as_text(save_path)):
+    if not os.path.exists(compat.as_text(save_path)):
```

Comment:
I'd use [gfile.Exists](https://github.com/tensorflow/tensorflow/blob/d23b8115e5db07805306ebbbf08c588b86e9eb27/tensorflow/python/platform/gfile.py#L26) (gfile is already imported in this file). That'll work with file systems registered with TensorFlow. You'd also need to append ".index" for v2 checkpoints, and we probably still need to support v1 checkpoints...

If you just want to get rid of the deprecation warning, it's probably easier to make a non-deprecated internal-to-TF version and use that here (you can also call it from the deprecated public endpoint).

Reply:
Thanks @allenlavoie for your suggestion, I just made some changes on `checkpoint_management.py` as you have mentioned.